language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -344,6 +351,33 @@ pub(crate) fn explain_publish_error(
             }
             println!("Re-run with --ignore-breaking-changes to publish anyway.")
         }
+        VMStatus::Error(MISSING_DEPENDENCY) => {
+            let err_indices = error_clone.indices();
+            let mut diags = Diagnostics::new();
+            for (ind_kind, table_ind) in err_indices {
+                if let IndexKind::FunctionHandle = ind_kind {
+                    let native_function = &(module.function_defs())[*table_ind as usize];
+                    let fh = module.function_handle_at(native_function.function);
+                    let mh = module.module_handle_at(fh.module);
+                    let function_source_map =
+                        src_map.get_function_source_map(FunctionDefinitionIndex(*table_ind));
+                    if let Ok(map) = function_source_map {
+                        let err_string = format!(
+                            "Missing implementation for the native function {}::{}",
+                            module.identifier_at(mh.name).as_str(),
+                            module.identifier_at(fh.name).as_str()
+                        );
+                        let diag = Diagnostic::new(
+                            diagnostics::codes::Declarations::InvalidFunction,
+                            (map.definition_location, err_string),
+                            Vec::<(Loc, String)>::new(),
```

Comment:
Nit: does this Vec::new need to have a type instantiation? I'm thinking it doesn't and the Rust can handle it without issue, but not certain.

Reply:
Thanks for reviewing the code, Tim! The Rust compiler seems to require the type annotation because it cannot infer type for type parameter `impl ToString` declared on the associated function `new`.